data/strings/strings.txt

```diff
@@ -25383,3 +25425,8 @@
     it = https://organicmaps.app/it/
     ru = https://organicmaps.app/ru/
     tr = https://organicmaps.app/tr/
+
+  [comma_separated_pair]
+    comment = Two values, separated by comma
```

Comment:
It would be better to mention where is it used. Do you have a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/126597376/230470854-6360ddf0-7806-4829-b1b3-e3fa1062102a.png)

Reply:
In this case we will get not nice translation for single track:
"1 tracks, 1 bookmarks".
However, if it is suitable, let us do this.

Reply:
You're right, the plural system doesn't support _two_ elements in one string when one can be plural, and another can not. Then let's leave it as is, but clarify in the comment, like "A number of tracks and number of bookmarks, separated by comma, like: 1 track, 5 bookmarks"